.github/workflows/ci.yaml

```diff
@@ -130,11 +130,65 @@ jobs:
           name: coverage-${{ matrix.python-version }}
           path: .coverage
 
+  tests-linux-alpha:
+    name: tests / run / ${{ matrix.python-version }} / Linux
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        python-version: ["3.11-dev"]
+    outputs:
+      python-key: ${{ steps.generate-python-key.outputs.key }}
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v3.0.1
+        with:
+          fetch-depth: 0
```

Comment:
Why set it to `0`? That will fetch the complete history which is unnecessary. A shallow copy (with the default `1`) should be enough. https://github.com/actions/checkout#usage